app/modules/transcribe/annotations/annotations.factory.js

```diff
@@ -57,8 +60,36 @@ function AnnotationsFactory(localStorageService) {
         return annotation;
     }
 
+
+    function checkVariants(annotation) {
+        //splits string by whitespace (different encodings)
+        var lemmas = annotation.text.split(/\s|&nbsp;/g);
+        var results = [];
```

Comment:
I don't know what it does, it's not returned or used anywhere else. Should it be there at all?